ang/civicase/case/details/people-tab/directives/case-roles-tab.directive.html

```diff
@@ -56,30 +56,15 @@ <h3>{{ ts('%1 Contacts', {1: roles.getCountOfAssignedRoles()}) }}</h3>
     </div>
   </div>
   <div class="civicase__people-tab__filter">
-    <paging
+    <civicase-paging
       class="paging-top"
-      page="$parent.rolesPage"
-      page-size="roles.ROLES_PER_PAGE"
-      total="roles.totalCount"
-      ng-show="roles.totalCount > roles.ROLES_PER_PAGE"
-      show-prev-next="true"
-      show-first-last="true"
-      text-first="{{ ts('<< First') }}"
-      text-last="{{ ts('Last >>') }}"
-      text-next="{{ ts('Next >') }}"
-      text-prev="{{ ts('< Prev') }}"
-    >
-    </paging>
-    <div>
-      <a
-        ng-repeat="letter in letters"
-        class="civicase__people-tab__filter-alpha-pager"
-        ng-class="{active: letter === rolesAlphaFilter}"
-        ng-click="setLetterFilter(letter, 'roles')">
-        {{ letter }}
-      </a>
-      <a href class="all" ng-click="setLetterFilter('', 'roles')">All</a>
-    </div>
+      paging-data="roles.pageObj"
+      paging-action="roles.goToPage($page)"
+    ></civicase-paging>
+    <civicase-relationship-letter-filter
+      model="rolesFilter"
+      click-listener="roles.filterRoles(rolesFilter)">
+    </civicase-relationship-letter-filter>
```

Comment:
why not use `ng-model` and `ng-change` ? we should try to follow the standard.
https://stackoverflow.com/questions/44875989/angularjs-custom-form-component-directive-using-ng-model